src/views/players/player/player_info.cpp

```diff
@@ -142,6 +150,7 @@ namespace big
 				ImGui::Text("Money In Bank: %d", stats.Money - stats.WalletBalance);
 				ImGui::Text("Total Money: %d", stats.Money);
 				ImGui::Text("Rank: %d (RP %d)", stats.Rank, stats.RP);
+				ImGui::Text("Health: %d (MaxHealth: % d)", ped_health, ped_maxhealth);
```

Comment:
Does this line work? There's a space between the `%` and the `d`.

Reply:
Yeah works in game, ill fix the spelling mistake when i can